openfisca_us/microdata/datasets/ce/ce.py

```diff
@@ -0,0 +1,280 @@
+import pkgutil
+
+from pandas import DataFrame, concat
+import numpy as np
+import h5py
+import yaml
+
+from openfisca_us.microdata.utils import US, dataset
+from openfisca_us.microdata.datasets.ce.raw_ce import RawCE
+
+
+@dataset
+class CE:
+    """OpenFisca-US dataset for the Consumer Expenditure Survey
+
+    The information extracted by RawCE and CE classes is from the "Interview
+    Survey" (https://www.bls.gov/cex/pumd-getting-started-guide.htm#section3).
+    Though there is also a "Diary Survey," the interview survey contains
+    quarterly household level (technically "consumer unit") data deemed most
+    appropriate for this project's needs. Only one set of files from the
+    Interview Survey, called "FMLI" is used, and contains quarterly
+    expenditures, annual income, assets, and liabilities, and household
+    characteristics and survey weights.
+    """
+
+    name = "ce"
+    model = US
+    # Ratio of mean cash contributions to charity vs total cash contribs, 2016.
+    # www.bls.gov/opub/btn/volume-8/the-relationship-between-cash-contributions-pretax-income-and-age.htm
+    # Table 3. Average expenditures and cash contributions, by quintiles of
+    #     pretax income, Consumer Expenditure Surveys, 2016.
+    MEAN_CASH_CONTRIB_OUTSIDE_HOUSEHOLD_2016 = 2080.85
```

Comment:
Doesn't have to be in this PR but having the microdata in this repo will make these constants much nicer to store and access openfisca-style

Reply:
yep